.stylelintrc.js

```diff
@@ -1,11 +1,18 @@
 module.exports = {
   extends: ['stylelint-config-carbon'],
+  plugins: ['stylelint-plugin-carbon-tokens'],
   rules: {
     'max-nesting-depth': null,
     'scss/no-global-function-names': null,
     'csstools/use-logical': null,
     'scss/load-no-partial-leading-underscore': null,
     'scss/double-slash-comment-inline': null,
     'no-duplicate-selectors': null,
+
+    'carbon/layout-use': null,
+    'carbon/theme-use': null,
+    'carbon/type-use': null,
+    'carbon/motion-duration-use': null,
+    'carbon/motion-easing-use': null,
```

Comment:
Doesn’t this defeat the purpose of using `stylelint-plugin-carbon-tokens`? 😹

Reply:
I think we would prefer to err on the side of caution and disable lines in our examples/story-specific SCSS files than to enable our team to bypass the tokens. This helps us be a bit more intentional about disabling the token usage (e.g. onboarding components that use colors that are not currently available as theme tokens).

Reply:
oh yeah sorry i'm doing this in stages because i saw a large amount of errors so i'm just starting with updating the rules haha. next step is enabling these and fixing them 🫡 

> I think we would prefer to err on the side of caution and disable lines in our examples/story-specific SCSS files than to enable our team to bypass the tokens. This helps us be a bit more intentional about disabling the token usage (e.g. onboarding components that use colors that are not currently available as theme tokens).

agreed. removing the line specific ones in favor of consolidating the rules to the top of the files is definitely a short-term fix since i think long term i'd we'd like to see those removed entirely (if possible).